pipeline/workflow-builder.Dockerfile

```diff
@@ -50,6 +50,27 @@ RUN /home/kogito/launch/build-app.sh ./resources
 #=============================
 FROM registry.access.redhat.com/ubi9/openjdk-17:1.21-2
 
+# Install JEP
+USER root
+
+# Install build dependencies for JEP
+RUN microdnf install -y \
+    python3 \
+    python3-devel \
+    gcc \
+    gcc-c++ \
+    make \
+    libffi-devel \
+    && microdnf clean all
+
+# Build JEP native lib
+RUN pip3 install --no-cache-dir jep
+
+# Copy JEP lib
+RUN PY_VER=$(python3 -c 'import sys; print(f"{sys.version_info.major}.{sys.version_info.minor}")') \
+    && cp /usr/local/lib64/python${PY_VER}/site-packages/jep/libjep.so /usr/lib64
```

Comment:
could you only run those RUN if python is needed?
Maybe introduce an env var PYTHON and then
RUN if [[ "$PYTHON" = true ]] ; then <commands>  ; fi

Reply:
fixed